.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,56 @@
+name: Publish Release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  base-publish:
+    if: startsWith(github.event.head_commit.message, 'bump:')
```

Comment:
Was wondering whether this will work. I don't know what the head commit message will be when the trigger type is release.